style.css

```diff
@@ -6,8 +6,89 @@ body {
   margin: 0;
 }
 
+.desktop-menu {
+  display: none;
+}
+
+.navmenu1 {
+  display: none;
+}
+
+nav ul li a {
+  color: #fff;
+  text-decoration: none;
+}
+
+#overlay {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: #191765;
+  background-image: url(images/menu-2.svg), url(images/menu-1.svg);
+  background-repeat: no-repeat no-repeat;
+  background-position: top left, bottom right;
+}
+
+.nav1 {
+  width: 30px;
+  height: 30px;
+  padding: 17px 13px 17px 0;
+  box-sizing: content-box;
+  align-self: flex-end;
+  margin: 50px 0;
+}
+
+#overlay a {
+  font-size: 32px;
```

Comment:
- [ ] In the mobile nav menu for the close button please reduce the size of it a little bit and change the color to match the given design.
- [ ] For the nav links Kindly add the correct margins between the nav links because now it's bigger than the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/169029454-67c59717-2d62-4274-82b8-8ea5e6a0107f.png) | ![image](https://user-images.githubusercontent.com/68796232/169029380-933f95c3-8965-4d47-824c-b960a7b8ab2c.png)  |